### ISSUE TITLE
Generate a concise report / csv file conveniently listing the failures in make check

### ISSUE THREAD

**Issue by udi-speedb** [speedb-io/speedb#288]
When running `make check`, all of the existing ut-s are run in parallel.
A common scenario is for a developer working on a candidate branch to test his branch using make check. 
All of the unit tests must pass and always do on a branch that was approved to be merged into origin/main.
However, during development, multiple unit tests may be broken. Currently, the user gets a messy report which is not easy to read (I have pasted on below).

What would be most welcome is a concise report listing the specific tests that have failed in some convenient to read format.

> 
> Computer:jobs running/jobs completed/%of started jobs/Average seconds to complete
> ETA: 534s Left: 8869 AVG: 0.06s  local:16/2490/100%/0.1s Note: Google Test filter = DBTest2.SharedWriteBufferLimitAcrossDB
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBTest2
> [ RUN      ] DBTest2.SharedWriteBufferLimitAcrossDB
> ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f3083aec03b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f3083acb859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f3083ea3911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f3083eaf38c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f3083eaf3f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa6a9) [0x7f3083eaf6a9] ??	??:0	
> #6   ./db_test2(+0x947f4) [0x55f2174a87f4] ??	??:0	
> #7   ./db_test2(+0x2806ec) [0x55f2176946ec] ??	??:0	
> #8   ./db_test2(+0x1fc87c) [0x55f21761087c] ??	??:0	
> #9   ./db_test2(+0x30303a) [0x55f21771703a] ??	??:0	
> #10  ./db_test2(+0x30874b) [0x55f21771c74b] ??	??:0	
> #11  ./db_test2(+0x30d399) [0x55f217721399] ??	??:0	
> #12  ./db_test2(+0x30e885) [0x55f217722885] ??	??:0	
> #13  ./db_test2(+0x315bd0) [0x55f217729bd0] ??	??:0	
> #14  ./db_test2(+0x316193) [0x55f21772a193] ??	??:0	
> #15  ./db_test2(+0x606f23) [0x55f217a1af23] ??	??:0	
> #16  ./db_test2(+0x607153) [0x55f217a1b153] ??	??:0	
> #17  /lib/x86_64-linux-gnu/libstdc++.so.6(+0xd6de4) [0x7f3083edbde4] ??	??:0	
> #18  /lib/x86_64-linux-gnu/libpthread.so.0(+0x8609) [0x7f30842ec609] ??	??:0	
> #19  /lib/x86_64-linux-gnu/libc.so.6(clone+0x43) [0x7f3083bc8163] ??	??:0	
> Aborted
> ETA: 533s Left: 8864 AVG: 0.06s  local:16/2495/100%/0.1s Note: Google Test filter = DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/0
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs
> [ RUN      ] DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/0
> db/db_test2.cc:470: Failure
> Expected equality of these values:
>   GetNumberOfSstFilesForColumnFamily(db_, "nikitich")
>     Which is: 1
>   static_cast<uint64_t>(2)
>     Which is: 2
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  db/db_test2.cc:470: Failure
> Expected equality of these values:
>   GetNumberOfSstFilesForColumnFamily(db_, "nikitich")
>     Which is: 1
>   static_cast<uint64_t>(2)
>     Which is: 2
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f0b2e3cb03b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f0b2e3aa859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f0b2e782911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f0b2e78e38c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f0b2e78e3f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_rethrow+0x4d) [0x7f0b2e78e6fd] ??	??:0	
> #6   ./db_test2(+0x28ec43) [0x5640c58bec43] ??	??:0	
> #7   ./db_test2(+0x282e23) [0x5640c58b2e23] ??	??:0	
> #8   ./db_test2(+0xd8a3d) [0x5640c5708a3d] ??	??:0	
> #9   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f0b2e3ac0b3] ??	??:0	
> #10  ./db_test2(+0xff71e) [0x5640c572f71e] ??	??:0	
> Aborted
> ETA: 530s Left: 8854 AVG: 0.06s  local:16/2505/100%/0.1s Note: Google Test filter = DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/1
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs
> [ RUN      ] DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/1
> ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f18ccdc803b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f18ccda7859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f18cd17f911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f18cd18b38c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f18cd18b3f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa6a9) [0x7f18cd18b6a9] ??	??:0	
> #6   ./db_test2(+0x947f4) [0x5582f33ba7f4] ??	??:0	
> #7   ./db_test2(+0x2806ec) [0x5582f35a66ec] ??	??:0	
> #8   ./db_test2(+0x1fc87c) [0x5582f352287c] ??	??:0	
> #9   ./db_test2(+0x30303a) [0x5582f362903a] ??	??:0	
> #10  ./db_test2(+0x30874b) [0x5582f362e74b] ??	??:0	
> #11  ./db_test2(+0x30d399) [0x5582f3633399] ??	??:0	
> #12  ./db_test2(+0x30e885) [0x5582f3634885] ??	??:0	
> #13  ./db_test2(+0x315bd0) [0x5582f363bbd0] ??	??:0	
> #14  ./db_test2(+0x316193) [0x5582f363c193] ??	??:0	
> #15  ./db_test2(+0x606f23) [0x5582f392cf23] ??	??:0	
> #16  ./db_test2(+0x607153) [0x5582f392d153] ??	??:0	
> #17  /lib/x86_64-linux-gnu/libstdc++.so.6(+0xd6de4) [0x7f18cd1b7de4] ??	??:0	
> #18  /lib/x86_64-linux-gnu/libpthread.so.0(+0x8609) [0x7f18cd5c8609] ??	??:0	
> #19  /lib/x86_64-linux-gnu/libc.so.6(clone+0x43) [0x7f18ccea4163] ??	??:0	
> Aborted
> ETA: 530s Left: 8853 AVG: 0.06s  local:16/2506/100%/0.1s Note: Google Test filter = DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/2
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs
> [ RUN      ] DBTestSharedWriteBufferAcrossCFs/DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs/2
> ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  ./db/db_test_util.h:845: Failure
> Expected equality of these values:
>   expected_flush_reason.load()
>     Which is: 4-byte object <05-00 00-00>
>   flush_job_info.flush_reason
>     Which is: 4-byte object <0E-00 00-00>
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f39765d203b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f39765b1859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f3976989911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f397699538c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f39769953f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa6a9) [0x7f39769956a9] ??	??:0	
> #6   ./db_test2(+0x947f4) [0x55cc0d36f7f4] ??	??:0	
> #7   ./db_test2(+0x2806ec) [0x55cc0d55b6ec] ??	??:0	
> #8   ./db_test2(+0x1fc87c) [0x55cc0d4d787c] ??	??:0	
> #9   ./db_test2(+0x30303a) [0x55cc0d5de03a] ??	??:0	
> #10  ./db_test2(+0x30874b) [0x55cc0d5e374b] ??	??:0	
> #11  ./db_test2(+0x30d399) [0x55cc0d5e8399] ??	??:0	
> #12  ./db_test2(+0x30e885) [0x55cc0d5e9885] ??	??:0	
> #13  ./db_test2(+0x315bd0) [0x55cc0d5f0bd0] ??	??:0	
> #14  ./db_test2(+0x316193) [0x55cc0d5f1193] ??	??:0	
> #15  ./db_test2(+0x606f23) [0x55cc0d8e1f23] ??	??:0	
> #16  ./db_test2(+0x607153) [0x55cc0d8e2153] ??	??:0	
> #17  /lib/x86_64-linux-gnu/libstdc++.so.6(+0xd6de4) [0x7f39769c1de4] ??	??:0	
> #18  /lib/x86_64-linux-gnu/libpthread.so.0(+0x8609) [0x7f3976dd2609] ??	??:0	
> #19  /lib/x86_64-linux-gnu/libc.so.6(clone+0x43) [0x7f39766ae163] ??	??:0	
> Aborted
> ETA: 334s Left: 6019 AVG: 0.06s  local:16/5340/100%/0.1s Note: Google Test filter = DBWriteBufferManagerTest/DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB/0
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBWriteBufferManagerTest/DBWriteBufferManagerTest
> [ RUN      ] DBWriteBufferManagerTest/DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB/0
> db/db_write_buffer_manager_test.cc:762: Failure
> Value of: s2
>   Actual: false
> Expected: true
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  db/db_write_buffer_manager_test.cc:762: Failure
> Value of: s2
>   Actual: false
> Expected: true
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f91953b803b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f9195397859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f919576f911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f919577b38c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f919577b3f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_rethrow+0x4d) [0x7f919577b6fd] ??	??:0	
> #6   ./db_write_buffer_manager_test(+0x15f2b3) [0x5584bbeb82b3] ??	??:0	
> #7   ./db_write_buffer_manager_test(+0x152f73) [0x5584bbeabf73] ??	??:0	
> #8   ./db_write_buffer_manager_test(+0xa4c4d) [0x5584bbdfdc4d] ??	??:0	
> #9   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f91953990b3] ??	??:0	
> #10  ./db_write_buffer_manager_test(+0xc6f5e) [0x5584bbe1ff5e] ??	??:0	
> Aborted
> ETA: 334s Left: 6016 AVG: 0.06s  local:16/5343/100%/0.1s Note: Google Test filter = DBWriteBufferManagerTest/DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB/1
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBWriteBufferManagerTest/DBWriteBufferManagerTest
> [ RUN      ] DBWriteBufferManagerTest/DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB/1
> db/db_write_buffer_manager_test.cc:767: Failure
> Expected equality of these values:
>   w_slowdown_set.size()
>     Which is: 3
>   num_dbs / 2
>     Which is: 2
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  db/db_write_buffer_manager_test.cc:767: Failure
> Expected equality of these values:
>   w_slowdown_set.size()
>     Which is: 3
>   num_dbs / 2
>     Which is: 2
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f72dff3b03b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f72dff1a859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f72e02f2911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f72e02fe38c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f72e02fe3f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_rethrow+0x4d) [0x7f72e02fe6fd] ??	??:0	
> #6   ./db_write_buffer_manager_test(+0x15f2b3) [0x556f6b2ed2b3] ??	??:0	
> #7   ./db_write_buffer_manager_test(+0x152f73) [0x556f6b2e0f73] ??	??:0	
> #8   ./db_write_buffer_manager_test(+0xa4c4d) [0x556f6b232c4d] ??	??:0	
> #9   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f72dff1c0b3] ??	??:0	
> #10  ./db_write_buffer_manager_test(+0xc6f5e) [0x556f6b254f5e] ??	??:0	
> Aborted
> ETA: 334s Left: 6013 AVG: 0.06s  local:16/5346/100%/0.1s Note: Google Test filter = DBWriteBufferManagerTest/DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB/0
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBWriteBufferManagerTest/DBWriteBufferManagerTest
> [ RUN      ] DBWriteBufferManagerTest/DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB/0
> db/db_write_buffer_manager_test.cc:289: Failure
> Expected equality of these values:
>   num_dbs + 1
>     Which is: 4
>   wait_count_db
>     Which is: 3
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  db/db_write_buffer_manager_test.cc:289: Failure
> Expected equality of these values:
>   num_dbs + 1
>     Which is: 4
>   wait_count_db
>     Which is: 3
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7f0a0dd7f03b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7f0a0dd5e859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7f0a0e136911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7f0a0e14238c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7f0a0e1423f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_rethrow+0x4d) [0x7f0a0e1426fd] ??	??:0	
> #6   ./db_write_buffer_manager_test(+0x15f2b3) [0x559660a722b3] ??	??:0	
> #7   ./db_write_buffer_manager_test(+0x152f73) [0x559660a65f73] ??	??:0	
> #8   ./db_write_buffer_manager_test(+0xa4c4d) [0x5596609b7c4d] ??	??:0	
> #9   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f0a0dd600b3] ??	??:0	
> #10  ./db_write_buffer_manager_test(+0xc6f5e) [0x5596609d9f5e] ??	??:0	
> Aborted
> ETA: 333s Left: 6010 AVG: 0.06s  local:16/5349/100%/0.1s Note: Google Test filter = DBWriteBufferManagerTest/DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB/1
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from DBWriteBufferManagerTest/DBWriteBufferManagerTest
> [ RUN      ] DBWriteBufferManagerTest/DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB/1
> db/db_write_buffer_manager_test.cc:289: Failure
> Expected equality of these values:
>   num_dbs + 1
>     Which is: 4
>   wait_count_db
>     Which is: 2
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  db/db_write_buffer_manager_test.cc:289: Failure
> Expected equality of these values:
>   num_dbs + 1
>     Which is: 4
>   wait_count_db
>     Which is: 2
> Received signal 6 (Aborted)
> #0   /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb) [0x7fa7c26bf03b] ??	??:0	
> #1   /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b) [0x7fa7c269e859] ??	??:0	
> #2   /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x7fa7c2a76911] ??	??:0	
> #3   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x7fa7c2a8238c] ??	??:0	
> #4   /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x7fa7c2a823f7] ??	??:0	
> #5   /lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_rethrow+0x4d) [0x7fa7c2a826fd] ??	??:0	
> #6   ./db_write_buffer_manager_test(+0x15f2b3) [0x56357e0892b3] ??	??:0	
> #7   ./db_write_buffer_manager_test(+0x152f73) [0x56357e07cf73] ??	??:0	
> #8   ./db_write_buffer_manager_test(+0xa4c4d) [0x56357dfcec4d] ??	??:0	
> #9   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fa7c26a00b3] ??	??:0	
> #10  ./db_write_buffer_manager_test(+0xc6f5e) [0x56357dff0f5e] ??	??:0	
> Aborted
> ETA: 14s Left: 394 AVG: 0.04s  local:16/10965/100%/0.0s Note: Google Test filter = WriteBufferManagerTest.CacheCost
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from WriteBufferManagerTest
> [ RUN      ] WriteBufferManagerTest.CacheCost
> memtable/write_buffer_manager_test.cc:153: Failure
> Value of: wbf->ShouldFlush()
>   Actual: false
> Expected: true
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  memtable/write_buffer_manager_test.cc:153: Failure
> Value of: wbf->ShouldFlush()
>   Actual: false
> Expected: true
> Aborted
> ETA: 14s Left: 391 AVG: 0.04s  local:16/10968/100%/0.0s Note: Google Test filter = WriteBufferManagerTest.ShouldFlush
> [==========] Running 1 test from 1 test case.
> [----------] Global test environment set-up.
> [----------] 1 test from WriteBufferManagerTest
> [ RUN      ] WriteBufferManagerTest.ShouldFlush
> memtable/write_buffer_manager_test.cc:50: Failure
> Value of: wbf->ShouldFlush()
>   Actual: false
> Expected: true
> terminate called after throwing an instance of 'testing::internal::GoogleTestFailureException'
>   what():  memtable/write_buffer_manager_test.cc:50: Failure
> Value of: wbf->ShouldFlush()
>   Actual: false
> Expected: true
> Aborted
> ETA: 0s Left: 0 AVG: 0.08s  local:0/11359/100%/0.1s s   
> Seq	Host	Starttime	JobRuntime	Send	Receive	Exitval	Signal	Command
> 2482	:	1670387566.033	     0.426	0	2088	134	0	./t/run-db_test2-DBTest2.SharedWriteBufferLimitAcrossDB >& t/log-run-db_test2-DBTest2.SharedWriteBufferLimitAcrossDB || bash -c "cat t/log-run-db_test2-DBTest2.SharedWriteBufferLimitAcrossDB; exit $?"
> 2501	:	1670387566.317	     0.197	0	1678	134	0	./t/run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-0 >& t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-0 || bash -c "cat t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-0; exit $?"
> 2502	:	1670387566.421	     0.269	0	2258	134	0	./t/run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-1 >& t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-1 || bash -c "cat t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-1; exit $?"
> 2503	:	1670387566.428	     0.272	0	2258	134	0	./t/run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-2 >& t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-2 || bash -c "cat t/log-run-db_test2-DBTestSharedWriteBufferAcrossCFs-DBTestSharedWriteBufferAcrossCFs.SharedWriteBufferAcrossCFs-2; exit $?"
> 5340	:	1670387712.663	     0.415	0	1550	134	0	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-0 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-0 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-0; exit $?"
> 5341	:	1670387712.693	     0.434	0	1672	134	0	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-1 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-1 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsMultipleDB-1; exit $?"
> 5348	:	1670387712.869	     0.276	0	1656	134	0	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-0 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-0 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-0; exit $?"
> 5349	:	1670387712.937	     0.226	0	1656	134	0	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-1 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-1 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB-1; exit $?"
> 10974	:	1670387828.118	     0.023	0	581	134	0	./t/run-write_buffer_manager_test-WriteBufferManagerTest.CacheCost >& t/log-run-write_buffer_manager_test-WriteBufferManagerTest.CacheCost || bash -c "cat t/log-run-write_buffer_manager_test-WriteBufferManagerTest.CacheCost; exit $?"
> 10977	:	1670387828.129	     0.022	0	583	134	0	./t/run-write_buffer_manager_test-WriteBufferManagerTest.ShouldFlush >& t/log-run-write_buffer_manager_test-WriteBufferManagerTest.ShouldFlush || bash -c "cat t/log-run-write_buffer_manager_test-WriteBufferManagerTest.ShouldFlush; exit $?"
> 5342	:	1670387712.727	   601.243	0	0	-1	15	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-0 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-0 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-0; exit $?"
> 5343	:	1670387712.762	   601.210	0	0	-1	15	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-1 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-1 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.MixedSlowDownOptionsSingleDB-1; exit $?"
> 5344	:	1670387712.796	   601.177	0	0	-1	15	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedBufferAcrossCFs1-0 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedBufferAcrossCFs1-0 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedBufferAcrossCFs1-0; exit $?"
> 5346	:	1670387712.830	   601.144	0	0	-1	15	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferAcrossCFs2-0 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferAcrossCFs2-0 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferAcrossCFs2-0; exit $?"
> 5351	:	1670387712.992	   600.983	0	0	-1	15	./t/run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB1-1 >& t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB1-1 || bash -c "cat t/log-run-db_write_buffer_manager_test-DBWriteBufferManagerTest-DBWriteBufferManagerTest.SharedWriteBufferLimitAcrossDB1-1; exit $?"
> make: *** [Makefile:985: check_0] Error 1
>

**Comment by udi-speedb** [comment]
A report such as this would be welcome:
![Screenshot from 2022-12-07 08-41-21](https://user-images.githubusercontent.com/106253580/206107814-54828e82-33ee-4dd3-9068-33cd768dd6ae.png)
